src/fragment-components/accordion/a-accordion-item.tsx

```diff
@@ -71,15 +71,15 @@ export const AAccordionItem = ({
 		)
 	});
 
-	return (<>
+	return (
 		<Adder
 		active={selected}
 		topAction={() => addAccordionItem()}
 		bottomAction={() => addAccordionItem(1)}>
 			<AComponent
 			componentObj={componentObj}
-			{...rest}
-			selected={selected}>
+			selected={selected}
```

Comment:
Not sure that the rest won't override selected if it's after. Could that be why selected is before?